### PR TITLE
initial migrations

### PR DESCRIPTION
**Pull request by Bskarol**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 source 'https://rubygems.org'
+gem 'rake'
 gem 'rspec-rails'
-
+gem 'sinatra'
+gem 'sinatra-activerecord'
```
Comment (kyjoya) [inline Gemfile]: We can remove these gems as we're not using Sinatra.

File: db/migrate/20170628184021_create_races.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateRaces < ActiveRecord::Migration[5.1]
+  def change
+    create_table :races do |t|
+      t.string :name, null: false, defualt: ""
+      t.string :location, null: false, default: ""
+      t.string :description, null: false, default: ""
+      t.string :date, null: false, default: ""
```
Comment (kyjoya) [inline db/migrate/20170628184021_create_races.rb]: Let's remove all of the `default: ""`'s on these two migrations.

**Comment by kyjoya** [comment]
@Bskarol once you address the comments in the PR and push again — we'll be ready to migrate!

File: Rakefile [hunk 3]
```diff
@@ -4,3 +4,12 @@
...
 Rails.application.load_tasks
+
+# Rakefile
+require 'sinatra/activerecord/rake'
+
+namespace :db do
+  task :load_config do
+    require './server'
+  end
+end
```
Comment (Bskarol) [inline Rakefile]: remove that

**Pull request merged**